pom.xml

```diff
@@ -217,6 +217,8 @@
                             <!-- if `mvn clean` is not executed and `mvn test` is interrupted, then perTestInputData may be not deleted -->
                             <exclude>src/test/resources/perTestInputData</exclude>
                             <exclude>src/test/resources/*.json</exclude>
+                            <exclude>NOTICE-binary</exclude>
+                            <exclude>LICENSE-binary</exclude>
```

Comment:
```suggestion
                            <exclude>License-binary</exclude>
```

Reply:
I think current file name is actually `LICENSE-binary`...

Reply:
this is because I changed that :)))